packages/features/auth/PermissionContainer.tsx

```diff
@@ -19,7 +19,8 @@ export const PermissionContainer: FC<AdminRequiredProps> = ({
   const session = useSession();
 
   // Admin can do everything
-  if (session.data?.user.role !== roleRequired || !UserPermissionRole.ADMIN) return null;
+  if (session.data?.user.role !== roleRequired && session.data?.user.role != UserPermissionRole.ADMIN)
```

Comment:
@sean-brydon Fixed this check.
cc @Udit-takkar. Please test this too.

Reply:
nice catch